openshift/release/generate-release.sh

```diff
@@ -14,3 +14,6 @@ else
 fi
 
 generate_pipeline_resources $output_file $image_prefix $tag
+
+# Update value for pipeline.tekton.dev/release label from "devel" to $tag 
```

Comment:
@waveywaves could you add a `if [[ -n ${tag} ]];then` check before the sed command
so that the label is not empty when the tag is `""`